libs/src/services/wormhole/ProxyWormhole.ts

```diff
@@ -0,0 +1,118 @@
+import type { Hedgehog } from '@audius/hedgehog'
+import type { EthContracts } from '../ethContracts'
+import type { EthWeb3Manager } from '../ethWeb3Manager'
+import type { IdentityService } from '../identity'
+import type { SolanaWeb3Manager } from '../solana'
+
+import bs58 from 'bs58'
```

Comment:
Does bs58 cause issues for us? I forget

Reply:
I don't think so, we have it in common already